Classes/common/CDTFetchChanges.h

```diff
@@ -0,0 +1,163 @@
+//
+//  CDTFetchChanges.h
+//  
+//
+//  Created by Michael Rhodes on 31/03/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@class CDTDatastore;
+@class CDTDocumentRevision;
+
+/**
+ Read changes to a database.
+ 
+ CDTFetchChanges reads the changes to a database from a given sequence value. It provides
+ callbacks for updated and created documents (documentChangedBlock) and deleted documents
+ (documentWithIDWasDeletedBlock). It also provides a completion block for when all changes
+ have been received and processed.
+ 
```

Comment:
The standard TD_Datastore changes feed isn't publicly exposed, this is designed to be. So I think discussing the differences vs. the internal changes feed isn't applicable here.

I think you're right about expanding on sequence value and so on, however.